testData/solutions/solutionsDescription/org_chrome_cloud4chrome.md

```diff
@@ -0,0 +1,46 @@
+# Cloud4chrome
```

Comment:
This file looks like a duplicate - this shouldn't be part of the pull request, is that correct?